compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/CodeAnalyzer.java

```diff
@@ -3950,9 +3950,11 @@ public void visit(BLangLimitClause limitClause) {
 
     @Override
     public void visit(BLangTypeTestExpr typeTestExpr) {
-        analyzeNode(typeTestExpr.expr, env);
-        BType exprType = typeTestExpr.expr.getBType();
-        BType typeNodeType = typeTestExpr.typeNode.getBType();
+        BLangExpression expr = typeTestExpr.expr;
+        BLangType typeNode = typeTestExpr.typeNode;
```

Comment:
Let's move this closer to where it is first used.

Reply:
Done